package.json

```diff
@@ -16,7 +16,6 @@
   ],
   "workspaces": [
     "examples/*/*",
-    "dev-modules/*",
```

Comment:
Ah, this was the critical problem, I assume, that mysteriously linked everything together...

Reply:
I'm hopeful that `ocular-publish` will ignore these packages now, but if not we will see errors the next time we try to do a monorepo publish, just a heads up! (Not sure how to safely test that step ahead of time...)